src/utils/getInfoLinkByChainId.ts

```diff
@@ -0,0 +1,20 @@
+import { SupportedChainId } from 'constants/chains'
+
+export default function getInfoLinkByChainId(chainId?: number) {
```

Comment:
have you thought about making this a const object instead? I'd love to extend this actually with the subgraph url too. Would be easier to extend if it was an object.

Reply:
Yeah I could definitely see putting this in the chain constants file as an object